vmdb/lib/token_manager.rb

```diff
@@ -28,7 +22,13 @@ def self.class_initialize(name = DEFAULT_NS, options = {})
   end
 
   def self.global_token_store(namespace)
-    @global_token_store[namespace] ||= {}
+    @token_caches[namespace] ||= begin
+      memcache_server = VMDB::Config.new("vmdb").config[:session][:memcache_server] || "127.0.0.1:11221"
+      Dalli::Client.new(memcache_server,
+                        :namespace  => "MIQ:VMDB:TOKENS:#{namespace.upcase}",
```

Comment:
token is not specific to rest api, at least that's the intent. The namespace parameter as passed in by the rest api is its module name "api", so the memcache namespace ends up being MIQ:VMDB:TOKENS:API.